package.json

```diff
@@ -45,6 +45,7 @@
   "resolutions": {
     "ansi-regex": "^5.0.1",
     "async": "^3.2.3",
+    "decode-uri-component": "^0.2.1",
```

Comment:
Though good to have, a simple bootstrap should also update the lock file. Thoughts?

Reply:
Okay to leave it as is.